torchvision/prototype/features/_image.py

```diff
@@ -77,7 +78,9 @@ def guess_color_space(data: torch.Tensor) -> ColorSpace:
             return ColorSpace.OTHER
 
     def show(self) -> None:
+        # This is a nice to have, but not a necessary method, for this early in the prototype
```

Comment:
Necessary no, but really helpful for debugging the dataset prototypes. Given that this is just one line, I would keep it in instead of adding it later unless you are strongly against it.

Reply:
If it assists your development it's fine to keep. See my above comment for all other cases of nice-to-haves that cause issues.